packages/expo-sms/src/SMS.ts

```diff
@@ -1,21 +1,23 @@
-import { Platform } from '@unimodules/core';
-import ExpoSMS from './ExpoSMS';
+import { UnavailabilityError } from '@unimodules/core';
 
-type SMSResponse = {
-  result: 'unknown' | 'sent' | 'cancelled';
-};
+import ExpoSMS from './ExpoSMS';
+import { SMSResponse } from './SMS.types';
 
 export async function sendSMSAsync(
   addresses: string | string[],
   message: string
 ): Promise<SMSResponse> {
   const finalAddresses = Array.isArray(addresses) ? addresses : [addresses];
   if (!ExpoSMS.sendSMSAsync) {
-    throw new Error(`SMS.sendSMSAsync is not supported on ${Platform.OS}`);
+    throw new UnavailabilityError(`expo-sms`, `sendSMSAsync`);
   }
   return ExpoSMS.sendSMSAsync(finalAddresses, message);
 }
 
+/**
+ * The device has a telephony radio with data communication support.
```

Comment:
```suggestion
/**
 * Returns whether the device has the capability to send SMS messages, which typically depends on
 * whether it has a cell radio or, in iOS's case, can use iMessage.
 *
 * This method always returns `false` in the iOS simulator and on web.
 */
```

is this bit about iMessage true? if you have a wifi iPad, is SMS "available"? technically the answer is no but it can still use iMessage, so we should (a) figure out what our library returns and (b) document these details carefully.

Reply:
I just copied the info from the docs...